src/cosmic/cosm.rs

```diff
@@ -58,8 +58,8 @@ pub enum LightTimeCalc {
     None,
     /// Accounts for light-time correction. This is corresponds to CN in SPICE.
     LightTime,
-    /// Accounts for light-time and stellar abberation where the solar system barycenter is the inertial frame. Corresponds to CN+S in SPICE.
-    Abberation,
+    /// Accounts for light-time and stellar aberration where the solar system barycenter is the inertial frame. Corresponds to CN+S in SPICE.
+    Aberration,
```

Comment:
This would cause a breaking change, requiring a new major version of Nyx. I'm currently working on that new version, but it's likely several months away. As such, if this PR is still to be merged against `master`, the typo cannot be fixed in the enum, only in the documentation.

Reply:
Resolved in commit 96a6ad32a5d0dc19579112d430d2411efecbbf96.